apps-rendering/src/components/shared/relatedContent.tsx

```diff
@@ -88,12 +88,10 @@ const listStyles = css`
 const styles = css`
 	border-top: 1px solid ${neutral[46]};
 	padding-top: ${remSpace[3]};
-	padding-left: ${remSpace[4]};
-	padding-right: ${remSpace[4]};
 
 	${until.wide} {
-		margin-left: -${remSpace[4]};
-		margin-right: -${remSpace[4]};
+		padding-left: ${remSpace[4]};
+		padding-right: ${remSpace[4]};
```

Comment:
FYI, there's a shorthand for vertical & horizontal:

```suggestion
		padding: 0 ${remSpace[4]};
```

Reply:
This is intentional as I need `padding-top` to be defined at wider breakpoints. 

`padding-top` is defined outside `${until.wide}`.

![image](https://user-images.githubusercontent.com/57295823/151833918-6140f81f-4ce2-40cc-bbe2-8cf736d350e3.png)

I only need `padding-left` and `padding-right` to be applied at smaller breakpoints (mobile view) - they are defined within `${until.wide}`
![image](https://user-images.githubusercontent.com/57295823/151833723-ecfff37e-8acd-4182-8265-59f26bf96bcb.png)

Doing `padding: 0 ${remSpace[4]}` would get rid of `top-padding` at smaller breakpoints, but I need to keep it! So I think it makes sense to be explicit and use `padding-left` and `padding-right`, unless there's a more elegant solution...